### PR TITLE
Add Autodoc, Carwow, Veeam, Nextlane, TeamViewer, Dremio, Iterable, Workia, Paddle, Riskified, Tabby, Teya, Wallet and Restream

### PR DESCRIPTION
**Pull request by Rxfa**
Add

- Autodoc (Moved its biggest tech hub to Lisbon. See [here](https://leitor.jornaleconomico.pt/noticia/talento-portugues-trouxe-centro-tecnologico-da-autodoc-para-portugal))
- Carwow (Hiring for tech roles in Porto. See [here](https://www.carwow.co.uk/jobs#gref))
- Veeam (Hiring for tech roles in Lisbon. See [here](https://careers.veeam.com/vacancies?query=&department=&location=Portugal&state=all&city=all))
- Nextlane (hiring for tech roles in Lisbon. See [here](https://www.linkedin.com/jobs/search/?currentJobId=3780178492&f_C=100697506%2C54593&geoId=92000000&origin=COMPANY_PAGE_JOBS_CLUSTER_EXPANSION&originToLandingJobPostings=3780178492%2C3794120174%2C3776787002%2C3790060295%2C3790728705%2C3776759296%2C3789329659%2C3780116726%2C3780123080))
- TeamViewer (Hiring for tech roles in Porto. see [here](https://careers.teamviewer.com/search/?q=&q2=&alertId=&title=&location=PT))
- Dremio (Hiring for tech roles to work in their Lisbon office or remotely. See [here](https://www.dremio.com/careers/job-postings/))
- Iterable (Hiring for tech roles in Lisbon. See [here](https://iterable.com/careers/))
- Workia (Hiring for tech roles in Lisbon. See [here](https://www.linkedin.com/jobs/search/?currentJobId=4062557245&f_C=91594487&geoId=92000000&origin=COMPANY_PAGE_JOBS_CLUSTER_EXPANSION&originToLandingJobPostings=4062557245))
- Paddle (Hiring for tech roles in Lisbon. See [here](https://www.paddle.com/careers#open-positions))
- Riskified (Hiring for tech roles in Lisbon. See [here](https://www.riskified.com/careers/#positions))
- Tabby (Hiring for tech roles to work remotely from anywhere in Portugal. See [here](https://tabby.ai/en-AE/careers#careers))
- Teya (Hiring for tech roles in Lisbon and Porto. See [here](https://careers.smartrecruiters.com/Teya))
- Wallet (Hiring for tech roles to work remotely from anywhere in Portugal. See [here](https://jobs.ton.org/companies/wallet-2))
- Restream (Hiring for tech roles in Lisbon. See [here](https://restream.io/careers))

### CODE REVIEW

**Comment by vpereira01** [comment]
Should be added check:

- Autodoc :heavy_check_mark: 
- Carwow :heavy_check_mark: (reference sites did not had tech jobs in Portugal but linkedin/jobsites had them)
- Nextlane :heavy_check_mark: 
- Nitro :heavy_check_mark: 
- TeamViewer :heavy_check_mark: 
- Dremio :heavy_check_mark: 
- Iterable :heavy_check_mark: 
- Veeam :heavy_check_mark: 
- Workia :white_check_mark: (seems only just a brand, the parent company as the same job opening https://apply.workable.com/equus-software/j/7A20413A8B/ )
- Paddle :heavy_check_mark: 
- Riskified :heavy_check_mark: 
- Tabby :heavy_check_mark: 
- Teya :heavy_check_mark: 
- Wallet :heavy_check_mark: 
- Restream :heavy_check_mark:

File: README.md [hunk 1]
```diff
@@ -44,12 +44,15 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
...
+| [Carwow](https://www.carwow.co.uk/) [:rocket:](https://www.carwow.co.uk/jobs#jobs-widget) | Online platform for comparing prices on new cars and offers. | `Porto` `Remote` |
 | [Critical TechWorks](https://www.criticaltechworks.com/) [:rocket:](https://join.criticaltechworks.com/jobs) | Developing next generation software systems for BMW. | `Braga` `Lisboa` <br> `Porto` |
 | [Daloop](https://daloop.io/) [:rocket:](https://daloop.recruitee.com/) | Data powered fleet electrification. | `Leça da Palmeira` |
 | [Mercedes-Benz.io](https://www.mercedes-benz.io) [:rocket:](https://www.mercedes-benz.io/jobs) | Shaping the digital future of Mercedes-Benz. | `Lisboa` `Braga` `Remote` |
 | [miio](https://www.miio.pt) [:rocket:](https://www.miio.pt/careers) | Electric mobility digital solutions with fleet management. | `Remote` |
 | [Nexar](https://data.getnexar.com/) [:rocket:](https://nexar-1702298813.teamtailor.com/#jobs) | Automative data collection and intelligence. | `Porto` |
+| [Nextlane](https://www.nextlane.com/) [:rocket:](https://www.nextlane.com/career/) | Software solutions to optimize logistics and supply chain in the automotive industry. | `Lisboa` |
```
Comment (vpereira01) [inline README.md:55]:
```suggestion
| [Nextlane](https://www.nextlane.com/) [:rocket:](https://www.nextlane.com/career/) | Software solutions to optimize automotive logistics and supply chain. | `Lisboa` |
```

Reduces description to max allowed chars, 70.

File: README.md [hunk 2]
```diff
@@ -181,6 +184,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Docker](https://docker.com) [:rocket:](https://www.docker.com/careers/) [:octocat:](https://github.com/docker) | Container and k8s tooling for developers. | `Remote` |
 | [Gitlab](https://about.gitlab.com/) [:rocket:](https://about.gitlab.com/jobs/all-jobs/) | DevOps platform for source code management, CI/CD, and collaboration. | `Remote` |
 | [Hugging Face](https://huggingface.co/) [:rocket:](https://apply.workable.com/huggingface/) [:octocat:](https://github.com/huggingface) | AI company specializing in natural language processing tools. | `Remote` |
+| [Nitro](https://www.gonitro.com/) [:rocket:](https://www.gonitro.com/about/careers) | Document productivity platform that simplifies PDF editing and collaboration. | `Porto` |
```
Comment (vpereira01) [inline README.md:187]:
```suggestion
| [Nitro](https://www.gonitro.com/) [:rocket:](https://www.gonitro.com/about/careers) | Productivity platform that simplifies PDF editing and collaboration. | `Porto` |
```

Reduces description to max allowed chars, 70.

File: README.md [hunk 3]
```diff
@@ -189,6 +193,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [SingleStore](https://www.singlestore.com/) [:rocket:](https://www.singlestore.com/careers/jobs/) | The scalable cloud database for data-intensive applications, deployed on AWS, Azure, or GCP. (formerly MemSQL) | `Lisboa` `Remote` |
 | [Synopsys](https://www.synopsys.com) [:rocket:](https://sjobs.brassring.com/TGnewUI/Search/Home/Home?partnerid=25235&siteid=5359) | Silicon chip design, verification, IP integration and application security | `Lisboa` `Porto` |
 | [Stream](https://getstream.io/) [:rocket:](https://getstream.io/team/#jobs) | API and SDK for chat messaging and activity feeds. | `Remote` |
+| [TeamViewer](https://www.teamviewer.com) [:rocket:](https://careers.teamviewer.com) | Remote access and support software for device management and collaboration. | `Porto` |
```
Comment (vpereira01) [inline README.md:196]:
```suggestion
| [TeamViewer](https://www.teamviewer.com) [:rocket:](https://careers.teamviewer.com) | Remote access, device management and collaboration. | `Porto` |
```

Reduces description to max allowed chars, 70.

File: README.md [hunk 4]
```diff
@@ -273,7 +280,9 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Tridonic](https://www.tridonic.com/com/en/) [:rocket:](https://career.zumtobelgroup.com/Jobs) | Smart and efficient lighting solutions. | `Porto` |
 | [Unit4](https://www.unit4.com/) [:rocket:](https://www.careers.unit4.com/) | ERP, Consulting. | `Algés` |
 | [VAKT Global Ltd.](https://www.vakt.com/) [:rocket:](https://www.vakt.com/careers/) | Enterprise grade blockchain commodities trading platform. | `Lisboa` |
+| [Veeam](https://www.veeam.com/) [:rocket:](https://careers.veeam.com/vacancies) | Backup, recovery, and data management solutions for businesses | `Lisboa` |
```
Comment (vpereira01) [inline README.md:283]:
```suggestion
| [Veeam](https://www.veeam.com/) [:rocket:](https://careers.veeam.com/vacancies) | Backup, recovery, and data management solutions for businesses. | `Lisboa` |
```
adding dot at the end.

File: README.md [hunk 5]
```diff
@@ -273,7 +280,9 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
...
 | [VOID Software](https://www.void.pt/) | Systems design & architecture using different technology stacks. | `Leiria` |
+| [Workia](https://workia.com/en/) | Collaboration platform that boosts workplace productivity and engagement. | `Lisboa` |
```
Comment (vpereira01) [inline README.md:285]:
```suggestion
| [Workia](https://workia.com/en/) | Simplifies the management of global employee mobility programs. | `Lisboa` |
```
Reduces description and better characterizes it

File: README.md [hunk 6]
```diff
@@ -299,17 +308,22 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
...
 | [Mastercard](https://www.mastercard.pt) [:rocket:](https://careers.mastercard.com)[:octocat:](https://github.com/mastercard) | Global payments company. | `Lisboa` |
 | [Moonpay](https://www.moonpay.com) [:rocket:](https://www.moonpay.com/careers) | Solutions for payments, smart contracts, and digital asset management. | `Lisboa` `Remote` |
-| [Pleo](https://www.pleo.io) [:rocket:](https://www.pleo.io/en/careers/jobs) | Smart payment cards and expense management tools for businesses | `Lisboa` `Remote` |
 | [Natixis](https://natixis.groupebpce.com/) [:rocket:](https://recrutement.natixis.com/) | French corporate and investment bank. | `Porto` |
 | [Onfido](https://www.onfido.com) [:rocket:](https://onfido.com/company/careers/) | Identity verification at the speed of life. | `Lisboa` |
+| [Paddle](https://www.paddle.com) [:rocket:](https://www.paddle.com/careers#open-positions) | Payment and subscription management platform for software vendors. | `Lisboa` |
+| [Pleo](https://www.pleo.io) [:rocket:](https://www.pleo.io/en/careers/jobs) | Smart payment cards and expense management tools for businesses | `Lisboa` `Remote` |
```
Comment (vpereira01) [inline README.md:314]: :+1:

File: README.md [hunk 7]
```diff
@@ -299,17 +308,22 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
...
+| [Riskified](https://www.riskified.com) [:rocket:](https://www.riskified.com/careers/#positions) | e-commerce fraud prevention. | `Lisboa` |
 | [Samlino Group](https://samlinogroup.com/) [:rocket:](https://samlinogroup.com/careers/) | Financial products comparison for several markets (e.g.: [ComparaJá](https://www.comparaja.pt)). | `Lisboa` |
 | [Seedrs](https://www.seedrs.com) [:rocket:](https://www.seedrs.com/careers) | Equity marketplace for everyone. | `Lisboa` |
 | [SIBS](https://www.sibs.com/) [:rocket:](https://sibs-sgps.zohorecruit.eu/careers) | Major Portuguese payment processor. | `Lisboa` |
 | [StudentFinance](https://studentfinance.com/) | Disrupting education through Income Share Agreements | `Lisboa` `Remote` |
 | [SwissBorg](https://swissborg.com/) [:rocket:](https://swissborg.com/careers) | Cryptocurrencies exchange | `Remote` |
 | [Switch](https://switchpayments.com/) [:rocket:](https://switchpayments.com/about/careers) | The payments infrastructure for interoperability. | `Lisboa` `Porto` |
+| [Tabby](https://tabby.ai/en-AE) | Buy now, pay later platform that offers flexible payment options. | `Remote` |
```
Comment (vpereira01) [inline README.md:323]:
```suggestion
| [Tabby](https://tabby.ai/en-AE) [:rocket:](https://tabby.ai/en-AE/careers#careers) | Buy now, pay later platform that offers flexible payment options. | `Remote` |
```

**Review by vpereira01 — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -242,6 +247,8 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Coverflex](https://www.coverflex.com/) [:rocket:](https://careers.coverflex.com/) | Payments and benefits management. | `Remote` |
 | [Deel](https://www.deel.com/) [:rocket:](https://jobs.ashbyhq.com/Deel?departmentId=8c27e7fa-f898-4de1-adf6-70c67c1e1c11&locationId=7a1f038b-147a-4a65-8ba5-d047fda72a10) [:octocat:](https://github.com/letsdeel) | Compliance, payroll, HR, and performance for any worker globally in a single SaaS platform. | `Remote` |
 | [Deimos](https://deimos-space.com/) [:rocket:](https://deimos-space.com/jobs/) | Advanced design solutions and turn-key space software systems. | `Lisboa` |
+| [Dremio](https://www.dremio.com/) [:rocket:](https://www.dremio.com/careers/job-postings/) | data-as-a-service platform for fast analytics and data integration. | `Lisboa` `Remote` |
+| [Elecnor Deimos](https://www.elecnor-deimos.com/) [:rocket:](https://www.elecnor-deimos.com/jobs/) | Advanced design solutions and turn-key space software systems. | `Lisboa` |
```
Comment (vpereira01) [inline README.md:251]:
```suggestion
```
clean up merge mess

**Review by vpereira01 — APPROVED** [review]
:+1:

File: README.md [hunk 9]
```diff
@@ -242,6 +247,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Coverflex](https://www.coverflex.com/) [:rocket:](https://careers.coverflex.com/) | Payments and benefits management. | `Remote` |
 | [Deel](https://www.deel.com/) [:rocket:](https://jobs.ashbyhq.com/Deel?departmentId=8c27e7fa-f898-4de1-adf6-70c67c1e1c11&locationId=7a1f038b-147a-4a65-8ba5-d047fda72a10) [:octocat:](https://github.com/letsdeel) | Compliance, payroll, HR, and performance for any worker globally in a single SaaS platform. | `Remote` |
 | [Deimos](https://deimos-space.com/) [:rocket:](https://deimos-space.com/jobs/) | Advanced design solutions and turn-key space software systems. | `Lisboa` |
+| [Dremio](https://www.dremio.com/) [:rocket:](https://www.dremio.com/careers/job-postings/) | data-as-a-service platform for fast analytics and data integration. | `Lisboa` `Remote` |
```
Comment (marmelo) [inline README.md:250]: I believe the right way is withouth hyphens.

```suggestion
| [Dremio](https://www.dremio.com/) [:rocket:](https://www.dremio.com/careers/job-postings/) | Data as a service platform for fast analytics and data integration. | `Lisboa` `Remote` |
```

File: README.md [hunk 10]
```diff
@@ -299,17 +308,22 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
...
-| [Pleo](https://www.pleo.io) [:rocket:](https://www.pleo.io/en/careers/jobs) | Smart payment cards and expense management tools for businesses | `Lisboa` `Remote` |
 | [Natixis](https://natixis.groupebpce.com/) [:rocket:](https://recrutement.natixis.com/) | French corporate and investment bank. | `Porto` |
 | [Onfido](https://www.onfido.com) [:rocket:](https://onfido.com/company/careers/) | Identity verification at the speed of life. | `Lisboa` |
+| [Paddle](https://www.paddle.com) [:rocket:](https://www.paddle.com/careers#open-positions) | Payment and subscription management platform for software vendors. | `Lisboa` |
+| [Pleo](https://www.pleo.io) [:rocket:](https://www.pleo.io/en/careers/jobs) | Smart payment cards and expense management tools for businesses. | `Lisboa` `Remote` |
 | [Revolut](https://www.revolut.com) [:rocket:](https://www.revolut.com/pt-PT/careers) [:octocat:](https://github.com/revolut-engineering) | A digital banking alternative to conventional banks. | `Lisboa` `Porto` |
+| [Riskified](https://www.riskified.com) [:rocket:](https://www.riskified.com/careers/#positions) | e-commerce fraud prevention. | `Lisboa` |
```
Comment (marmelo) [inline README.md:316]:
```suggestion
| [Riskified](https://www.riskified.com) [:rocket:](https://www.riskified.com/careers/#positions) | E-commerce fraud prevention. | `Lisboa` |
```

**Review by marmelo — CHANGES_REQUESTED** [review]

**Review by afonsojramos — APPROVED** [review]

**Comment by afonsojramos** [comment]
That's a lot of them! Thanks!

**Review by marmelo — APPROVED** [review]

**Comment by marmelo** [comment]
Thank you @Rxfa!

**Pull request merged**